src/ngx_http_lua_shdict.c

```diff
@@ -1363,8 +1392,21 @@ ngx_http_lua_shdict_incr(lua_State *L)
             return 2;
         }
 
-        /* add value */
-        num = value + init;
+
+        if (flags == NGX_HTTP_LUA_SHDICT_INCR) {
+            /* add value */
+            num = value + init;
+        } else /* decr */ {
```

Comment:
@p0pr0ck5 Note the position of the comment and also the extra blank line before `} else{` in my example above.